app/views/admin/preview.html.haml

```diff
@@ -1,9 +1,2 @@
 = render partial: 'shared/header'
-= QuestionnaireGenerator.generate_questionnaire(response_id: nil,
-  content: @questionnaire.content,
-  title: @questionnaire.title,
-  submit_text: 'Opslaan',
-  action: '/admin/preview_done',
-  unsubscribe_url: nil,
-  params: { authenticity_token: form_authenticity_token(form_options: { action: '/admin/preview_done',
-                                                                        method: 'post' }) })
+= @content
```

Comment:
Wat was trouwens de reden om deze logica naar de controller te halen? Je doet het niet voor de informed consent view namelijk.

Als ik heel eerlijk ben dan vind ik het wel meer "controller-logica" dwz het past beter in de controller omdat je ook dingen meegeeft zoals action urls en callback urls enz. die afhankelijk zijn van methode(namen) in de controller. Dus dat zou ervoor pleiten om ook de code voor de informed consent naar de controller te halen.